src/scripts/Providers/OSUI/Datepicker/Flatpickr/AbstractFlatpickr.ts

```diff
@@ -495,6 +495,9 @@ namespace Providers.OSUI.Datepicker.Flatpickr {
 			const isInputDisable = this.datePickerPlatformInputElem.disabled;
 			if (this.provider.isOpen === false && isInputDisable === false) {
 				this.provider.open();
+
+				// Focus on input element, after the open trigger occur
+				this.flatpickrInputElem.focus();
```

Comment:
I would change to `this.flatpickrInputElem?.focus()` to validate first if the element exist. We can be using the fixed option or the native one, without the input